package.json

```diff
@@ -24,8 +24,8 @@
     "xml": "^1.0.1"
   },
   "devDependencies": {
-    "jest": "^27.2.3",
-    "libxmljs2": "^0.29.0",
+    "jest": "^29",
+    "libxmljs2": "^0.33.0",
```

Comment:
maybe switch it out?

https://github.com/marudor/libxmljs2/commit/7ef018cfa3be3b908530e0cb4f3b6bdec6af6633

Reply:
There is no xsd validator in fast-xml-parser https://github.com/NaturalIntelligence/fast-xml-parser/issues/220